### PR TITLE
add the code template for quicksort and refactor

### LINKED ISSUE #5: Quick sort(Swap involved)

### PR DESCRIPTION
**Pull request by seanjyjy**
* Add code template, explanation and algorithm for quicksort
* Refactor files to be neater
* Abstract methods out to Util folder

### CODE REVIEW

File: src/visualizer/styles.css [hunk 1]
```diff
@@ -54,9 +54,11 @@
 
 .code {
   align-content: center;
+  border: 2px solid grey;
+  /*border: 2px solid grey;*/
```
Comment (seanjyjy) [inline src/visualizer/styles.css:58]: not sure why is there 57 and 58? my file doesn't contain it.

Comment (keanecjy) [reply src/visualizer/styles.css:58]: Why are there two styles.css files for sorting visualizer?

File: src/visualizer/styles.css [hunk 2]
```diff
@@ -1,66 +1,66 @@
...
 
 .speed-selector-box {
-  align-items: flex-end;
-  display: flex;
-  flex: 4;
-  flex-direction: column;
+    align-items: flex-end;
+    display: flex;
+    flex: 4;
+    flex-direction: column;
```
Comment (keanecjy) [inline src/visualizer/styles.css]: Indents should be kept at 2 for css files!

**Review by keanecjy — CHANGES_REQUESTED** [review]

File: src/visualizer/styles.css (hunk 1 above)
Comment (seanjyjy) [reply src/visualizer/styles.css:58]: oh wait i also dont know i will delete the one outside

**Review by keanecjy — APPROVED** [review]
LGTM. Great work.

**Pull request merged**